### PR TITLE
fix: button widths

### LINKED ISSUE #4133: Cancel buttons are too narrow in Japanese
### Self Checks

- [X] This is only for bug report, if you would like to ask a quesion, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### Dify version

0.6.6

### Cloud or Self Hosted

Cloud, Self Hosted (Docker)

### Steps to reproduce

1. Change your language to Japanese
2. Go to the application list page ( http://localhost/apps )
3. Edit or delete from the menu in the lower right corner of an existing app.

スクリーンショット 2024-05-07 11 06 34
スクリーンショット 2024-05-07 11 18 04

### ✔️ Expected Behavior

The cancel buttons are displayed without overflowing (no line breaks)

### ❌ Actual Behavior

_No response_

### PR DESCRIPTION
**Pull request by tomoyuki28jp**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #4133

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

1. Change your language to Japanese
2. Go to the application list page ( http://localhost:3000/apps )
3. Edit or delete from the menu in the lower right corner of an existing app.

# Suggested Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods
- [ ] `optional` I have made corresponding changes to the documentation 
- [ ] `optional` I have added tests that prove my fix is effective or that my feature works
- [ ] `optional` New and existing unit tests pass locally with my changes

# Screenshots
スクリーンショット 2024-05-07 16 48 01
スクリーンショット 2024-05-07 16 48 07

### CODE REVIEW

File: web/app/components/base/confirm-ui/index.tsx [hunk 1]
```diff
@@ -42,8 +42,8 @@ const ConfirmUI: FC<IConfirmUIProps> = ({
       </div>
 
       <div className='flex gap-3 mt-4 ml-12'>
-        <Button type='primary' onClick={onConfirm} className='flex items-center justify-center w-20 text-center text-white rounded-lg cursor-pointer h-9 '>{confirmText || t('common.operation.confirm')}</Button>
-        <Button onClick={onCancel} className='flex items-center justify-center w-20 text-center text-gray-500 border rounded-lg cursor-pointer h-9 border-color-gray-200'>{cancelText || t('common.operation.cancel')}</Button>
+        <Button type='primary' onClick={onConfirm} className='flex items-center justify-center w-21 text-center text-white rounded-lg cursor-pointer h-9 '>{confirmText || t('common.operation.confirm')}</Button>
```
Comment (zxhlyh) [inline web/app/components/base/confirm-ui/index.tsx]: Maybe we can use min-w-20 to replace it?

**Comment by TinsFox** [comment]
Hye @tomoyuki28jp 

I have a small suggestion here. If you modify it like this, you will still have this problem when using this component in other places.

It is better to deal with it in its underlying components to obtain the effect once and for all. We can easily fix it by just adding `whitespace-nowrap` to this line

https://github.com/langgenius/dify/blob/8137d63000fea6496e6184e30604e6f3967141c5/web/app/components/base/button/index.css#L5

The new code looks like this
```css
 .btn {
    @apply inline-flex justify-center items-center content-center h-9 leading-5 rounded-lg px-4 py-2 text-base cursor-pointer whitespace-nowrap;
  }
```

**Comment by tomoyuki28jp** [comment]
@TinsFox Thank you for your advice! I've also fixed the CSS styles.

**Review by zxhlyh — DISMISSED** [review]

**Comment by tomoyuki28jp** [comment]
@zxhlyh Thank you for your review, I've updated my code to use min-w-20.

**Review by zxhlyh — APPROVED** [review]
Thanks.

**Pull request merged**